pkgs/by-name/ne/netatalk/package.nix

```diff
@@ -13,29 +13,33 @@
   pam,
   perl,
   pkg-config,
+  meson,
+  ninja,
+  file,
   python3,
+  cracklib,
+  cups,
+  libtirpc,
+  openldap,
+  glib,
+  dbus,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "netatalk";
-  version = "3.1.19";
+  version = "4.1.1";
 
   src = fetchurl {
-    url = "mirror://sourceforge/netatalk/netatalk/netatalk-${finalAttrs.version}.tar.bz2";
-    hash = "sha256-p0pzHwnjNNWPsWRBflgaZB+ijMP5bF99TEdLs4mVKsI=";
+    url = "mirror://sourceforge/netatalk/netatalk/netatalk-${finalAttrs.version}.tar.xz";
+    hash = "sha256-LI0xIkWzmotzSsmo60URC8A94wiJKOV0MQqZDTuAAkE=";
   };
 
-  patches = [
-    ./000-no-suid.patch
-    ./001-omit-localstatedir-creation.patch
-  ];
-
   nativeBuildInputs = [
-    autoreconfHook
     pkg-config
+    meson
+    ninja
+    file
     perl
```

Comment:
Please also put a `perl` in buildInputs, it ends up being used at runtime.
If it is only in native inputs cross builds get the wrong perl.